src/scripts/main.js

```diff
@@ -21,7 +21,27 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  // Create a <ul> element
+  const ul = document.createElement('ul');
+
+  // Iterate over each key in the data object
+  Object.keys(data).forEach((key) => {
+    // Create an <li> element for each key
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    // If the current key has nested objects, recursively create a nested list
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
```

Comment:
This section of the code is responsible for checking whether the current key has nested objects and if it does, it calls the createTree function recursively. However, this recursion is not working as expected because you are not appending the result of the recursive call to the current list item. You should append the result of the createTree function to the list item before appending the list item to the list.